src/components/PageTitle/index.js

```diff
@@ -7,7 +7,7 @@ class PageTitle extends Component {
   componentDidUpdate(prevProps) {
     const { title, subTitle } = this.props
     if (title !== prevProps.title || subTitle !== prevProps.subTitle) {
-      this.container.focus()
+      this.container && this.container.focus()
```

Comment:
I think the change here is a reasonable change, since it checks for `this.container` first before calling `this.container.focus()` but I agree it seems unrelated. 
It's possible they were getting an error when focus is set too soon on `componentDidUpdate`. We'd have to clarify with the external integrator.